democracylab/settings.py

```diff
@@ -551,3 +551,5 @@ def read_connection_config(config):
 MALICIOUS_FWD_PATTERNS = os.environ.get("MALICIOUS_FWD_PATTERNS", None)
 
 PASSWORD_RESET_TIMEOUT_DAYS = os.environ.get("PASSWORD_RESET_TIMEOUT_DAYS", 7)
+
+SECRET_ADMIN = os.environ.get("SECRET_ADMIN", "")
```

Comment:
Let's default to normal \admin path, so the site won't break if a developer forgets to set this environment variable.
```suggestion
SECRET_ADMIN = os.environ.get("SECRET_ADMIN", "admin")
```